src/creeps/Node.h

```diff
@@ -52,7 +52,7 @@ class Node {
             const int pri = 0) : xPos(xPos), yPos(yPos), lv(lv), pri(pri) {}
 
     virtual ~Node() {
-        logv("destroy Node\n");
+        //logv("destroy Node\n");
```

Comment:
Logs should not be commented out
Either uncomment them, or remove them entirely
The core repo should not contain commented code